freeform_json_to_yaml.py

```diff
@@ -0,0 +1,440 @@
+        '''
+        inventory = self.json_data['availableInventory']
+        supplies = []
+        for item in inventory:
+            supply = {"type": SUPPLY_MAP[item]}
+            if type(inventory[item]) is int:
+                supply['quantity'] = inventory[item]
+            else:
+                if inventory[item]:
+                    supply["quantity"] = 1
+                    supply['reusable'] = True
+                else:
+                    supply['quantity'] = 0 
+            supplies.append(supply)
+        return supplies
+    
+
+    def get_characters(self):
+        '''
+        Returns a yaml-formatted object of characters based on the json data
+        '''
+        characters = self.json_data['patientDataList']
+        yaml_characters = []
+        for c in characters:
+            new_c = {}
+            new_c['id'] = c['name']
+            new_c['name'] = c['raceEthnicity']['firstName']
+            new_c['unstructured'] = ""
+            new_c['unstructured_postassess'] = ""
+            patient = c['patient']
+            race = c['raceEthnicity']['preset']
+
+            demographics = {
+                'age': randint(25, 35) if patient in ['Helga', 'Bob', 'Lily', 'Military Mike'] else randint(45, 58),
+                'sex': 'M' if patient in ['Gary', 'Military Mike', 'Bob'] else 'F',
+                'race': race.split(' ')[0] if 'Islander' not in race and 'Indian' not in race else race
+            }
+
+            is_military = False
+            us_military = True
+            for gear in c['militaryGear']:
+                if len(c['militaryGear'][gear]) > 0:
+                    is_military = True
+                    break
+            if c['clothing']['bottomColor'] != 'original' or c['clothing']['topColor'] != 'original':
+                us_military = False
+
+            if is_military:
+                demographics['military_disposition'] = 'Allied US' if us_military else 'Allied'
+                sim_env = self.json_data['scene'].split('-')[1].replace('sub', 'submarine')
+                if sim_env != 'submarine':
+                    demographics['role'] = 'Infantry'
+                demographics['mission_importance'] = 'normal'
+                if us_military:
+                    demographics['military_branch'] = 'US Army' if sim_env in ['jungle', 'desert', 'urban'] else 'US Navy'
+            
+            new_c['demographics'] = demographics
+            vitals = c['vitals']
+            new_c['vitals'] = {
+                'avpu': 'ALERT' if MENTAL_STATUS_MAP[vitals['mood']] != 'UNRESPONSIVE' else 'UNRESPONSIVE',
+                'ambulatory': c['triageStatus']['sort'] == 'walker',
+                'mental_status': MENTAL_STATUS_MAP[vitals['mood']],
+                'breathing': BREATHING_MAP[vitals['breath']],
+                'heart_rate': PULSE_MAP[vitals['pulse']],
+                'spo2': 67 if vitals['SpO2'] == 'none' else 85 if vitals['SpO2'] == 'low' else 95
```

Comment:
Now that ITM-476 (spo2 change) has been checked into development, I guess you can update this to use NORMAL/LOW/NONE?